deepchem/feat/material_featurizers/cgcnn_featurizer.py

```diff
@@ -0,0 +1,179 @@
+import os
+import json
+import numpy as np
+from typing import Tuple
+
+from deepchem.utils import download_url, get_data_dir
+from deepchem.utils.typing import PymatgenStructure
+from deepchem.feat import MaterialStructureFeaturizer
+from deepchem.feat.graph_data import GraphData
+
+# FIXME: it is better to add this json to DeepChem AWS
+ATOM_INIT_JSON_URL = 'https://raw.githubusercontent.com/txie-93/cgcnn/master/data/sample-regression/atom_init.json'
```

Comment:
Could I upload this json file....?

Reply:
Yes absolutely! I'll try to get AWS permissions fixed for you by tomorrow